util/fipstools/delocate/delocate.go

```diff
@@ -59,6 +59,30 @@ const (
 	aarch64
 )
 
+// represents a unique symbol for an occurrence of OPENSSL_ia32cap_P.
+type cpuCapUniqueSymbol struct {
+	registerName string
+	suffixUniqueness string
+}
+
+const cpuCapIntelSymbolPrefix = "LOPENSSL_ia32cap_P_"
```

Comment:
can we call it `x86` because it's used both on Intel and AMD?

Reply:
makes sense!